coveralls.json

```diff
@@ -9,6 +9,7 @@
     "test/support/channel_case.ex",
     "test/support/conn_case.ex",
     "web/web.ex",
-    "web/views/error_helpers.ex"
+    "web/views/error_helpers.ex",
+    "web/controllers/github_api/http_client.ex"
```

Comment:
I'm using a mock to test the Github API see https://github.com/dwyl/dwylbot/pull/54/files#diff-d9a1678446e1699ff1ed587b08c02c9f . So the mock replace the test of this file.